code/glTF2/glTF2Asset.inl

```diff
@@ -270,13 +270,14 @@ Ref<T> LazyDict<T>::Retrieve(unsigned int i)
         throw DeadlyImportError("GLTF: Object at index \"" + to_string(i) + "\" is not a JSON object");
     }
 
-    T* inst = new T();
+    // Unique ptr prevents memory leak in case of Read throws an exception
+    auto inst = std::unique_ptr<T>();
```

Comment:
Don't you need to actually put something in `unique_ptr`? It should default-construct as empty (effectively nullptr).

Reply:
you're right, thank you!